backend/config.py

```diff
@@ -0,0 +1,11 @@
+from dotenv import load_dotenv
```

Comment:
Did you add `dotenv` to the `requirements.txt`?

Reply:
froze the dotenv to the requirements.txt file